docs/events/gsoc/2024/gradle-build-server-android.md

```diff
@@ -110,10 +110,10 @@ As the popularity of Android development grows, the need for efficient build aut
 
 **Limitations:**
 
-- Android generates a lot of intermediate sources such as AIDL, Render script, C/CPP, etc. Unfortunately AGP doesn't provide any APIs to retrieve these files. R.jar is one of these intermediate sources which is crucial for Android development so as a workaround I have retrieved the file using the process task for the build variant however, in some cases the task may not be registered during the sync in which case the dependency for R.jar is not provided. ([Issue #181](https://github.com/microsoft/build-server-for-gradle/issues/181))
+- Android generates a lot of intermediate sources such as AIDL, Render script, C/CPP, etc. Unfortunately AGP doesn't provide any APIs to retrieve these files. `R.jar` is one of these intermediate sources which is crucial for Android development so as a workaround I have retrieved the file using the process task for the build variant however, in some cases the task may not be registered during the sync in which case the dependency for `R.jar` is not provided. ([Issue #181](https://github.com/microsoft/build-server-for-gradle/issues/181))
 - There various kinds of projects that come under Android development such as Application, Library, Instant App, Dynamic Modules, Android Test, etc. I have added support for the most commonly used projects - Application and Library but the current implementation may require further enhancements to support other kinds of projects. ([Issue #182](https://github.com/microsoft/build-server-for-gradle/issues/182))
-- My implementation takes into account any user defined build variants and the default build variants - debug and release except the default test variants - test and androidTest. ([Issue #183](https://github.com/microsoft/build-server-for-gradle/issues/183))
```

Comment:
@oleg-nenashev I have a PR for this issue in GBS. Should that be mentioned here? The mentioned issue does link to the PR though.

Reply:
Yes, though it's been nice to mention that it was after the project ended

Reply:
https://github.com/gradle/community/pull/75/commits/699ae329023fdf20c13894898ef14581b40041b3